streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java

```diff
@@ -152,11 +150,17 @@ public void shouldQueryOnlyActivePartitionStoresByDefault() throws Exception {
                 }
                 return true;
             } catch (final InvalidStateStoreException exception) {
-                assertThat(
-                    exception.getMessage(),
-                    containsString("Cannot get state store source-table because the stream thread is PARTITIONS_ASSIGNED, not RUNNING")
-                );
-                LOG.info("Streams wasn't running. Will try again.");
+                final String message = exception.getMessage();
+                final boolean exceptionNotRunning = message.startsWith("Cannot get state store source-table because the stream thread is PARTITIONS_ASSIGNED, not RUNNING");
```

Comment:
It might be more robust to use "contains" instead of "startsWith", but I won't insist on it.